sdcm/utils.py

```diff
@@ -196,6 +213,37 @@ def upload_file(cls, file_path):
             logger.debug("Unable to upload to S3: %s" % e)
             return ""
 
+    @classmethod
+    def download_files_by_testid_from_bucket(cls, bucket_name, testid):
+        filenames = cls.find_filename_by_testid(bucket_name, testid)
+        bucket = cls._get_bucket(bucket_name)
+        for f in filenames:
+            try:
+                bucket.download_file(Key=f, Filename=f)
+            except Exception as details:
+                logger.info(details)
+                subprocess.call('curl -o {} https://{}.s3.amazonaws.com/{}'.format(f, bucket_name, f), shell=True)
+
+
+def list_logs_by_test_id(test_id):
+    bucket_name = 'cloudius-jenkins-test'
```

Comment:
can the bucket name be defined in one place ?

and why it's still named cloudius ?

Reply:
2. I use current bucket which is store logs and data by job_name. Thinking about creation new one. But i  think, i don't have such permission.
1. if i understood you correctly, now it is set only in one place in function. It allow to use S3Storage class with different buckets. another way, it could be a part of S3Storage class.

Reply:
Or it could be some Global Var

Reply:
both are fine, so it would be easy to replace if we decide to move to a new bucket

Reply:
I think bucket name should be part of the `S3Storage` object

Reply:
Fixed